rllib/core/learner/learner.py

```diff
@@ -730,6 +748,21 @@ def get_state(self) -> Mapping[str, Any]:
         # TODO: once we figure out the optimizer format, we can set/get the state
         return {"module_state": self._module.get_state()}
 
+    @abc.abstractmethod
+    def _is_module_compatible_with_learner(self, module: RLModule) -> bool:
+        """Check whether the module is compatible with the learner.
+
+        Are they both torch or tf? If there is a random RLModule for example, it will
+        not be a torch or tf module. Therefore we should not consider it during
```

Comment:
"Are they both torch or tf?" is not clear to me.

Reply:
yep.